### DISCUSSION TITLE
Welcome to bash-it Discussions!

### DISCUSSION THREAD

**Discussion by NoahGorny** [Bash-it/bash-it#2177] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and be open-minded :)

You are welcome to say hi here in this thread and tell us how you use Bash-it 😄 

## Note
  This is not a replacement for our issue tracker. If you have a bug/feature request- please submit an issue.

## Kudos
Thanks for @EmilySeville7cfg for the interesting idea!

Noah

**Comment by seefood** [comment]
Hey Noah, Is bash-it still alive or are you too busy to maintain it? do you need a hand managing it? I see lots of PRs piling up for over a year now.
